hermes/views.py

```diff
@@ -78,19 +87,57 @@ def validate_brightness_unit(self, data):
             if row['brightness_unit'] not in brightness_units:
                 raise ValidationError(f'Unrecognized brightness unit. Accepted brightness units are {brightness_units}')
 
-
-class CandidateMessageSubSchema(Schema):
-    authors = fields.String()
-    main_data = fields.Nested(CandidateDataSchema)
-
-
-class CandidateMessageSchema(Schema):
+class MessageSchema(Schema):
     title = fields.String(required=True)
     topic = fields.String(required=True)
     event_id = fields.String()
     message_text = fields.String(required=True)
-    author = fields.String(required=True)
-    data = fields.Nested(CandidateMessageSubSchema)
+    submitter = fields.String(required=True)
+    authors = fields.String()
+    @property
+    @abstractmethod
+    def data(self):
+        pass
+
+
+class CandidateMessageSchema(MessageSchema):
+    data = fields.Nested(CandidateDataSchema)
+
+
+class PhotometryDataSchema(Schema):
+    target_name = fields.String()
```

Comment:
would it make more sense to require a `TargetTable` of some kind that included a valid name and relevant info like position, alternate names, or orbital elements?

Reply:
Worth a thought. I worry about the alternate names thing. That is such a can of worms. Orbital elements is another question. Could we request RA and Dec for sidereal and orbital elements for non-sidereal?

Reply:
That's easy for a target. I think it becomes an unnecessary burden for each filter.

Reply:
I agree we should support both RA/Dec and orbital elements.  For the alternative names, usually a few names are most important so we could offer the option of up to four name fields for example?  Or we could have just one field for names and allow slash separators e.g. OGLE-2022-BLG-0001/MOA-2022-BLG-023 ?

Reply:
I would argue a required name, and an optional `alternate_designations` field with a comma separated list (because comets (i.e. C/1995 O1) break everything).